### ISSUE TITLE
Paper specifications?  (a4/letter, page limit)

### ISSUE THREAD

**Issue by snoeyink** [JuliaCon/proceedings-review#32]
I'm trying to find the specifications of page limit and paper size for the proceedings.  
One author asked if it was ok to use single column, and another has formatted for A4 paper...
Thanks

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks.

For a list of things I can do to help you, just type:

```
@whedon commands
```

**What happens now?**

This submission is currently in a `pre-review` state which means we are waiting for an editor to be assigned and for them to find some reviewers for your submission. This may take anything between a few hours to a couple of weeks. Thanks for your patience :smile_cat:

You can help the editor by looking at [this list of potential reviewers](http://bit.ly/2V9GSFq) to identify individuals who might be able to review your submission (please start at the bottom of the list). Also, feel free to suggest individuals who are not on this list by mentioning their GitHub handles here.

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
PDF failed to compile for issue #32 with the following error: 

 Can't find any papers to compile :-(

**Comment by matbesancon** [comment]
There is a provided template http://github.com/JuliaCon/JuliaConSubmission.jl which should not be changed.
The page limit is one + references for extended abstracts and around 8 pages (there is no category or such on the issue)

**Comment by snoeyink** [comment]
The template declares letterpaper but sets paperheight and paperwidth of A4 with letter paper size.  
\DeclareOption{letterpaper}
   {\setlength\paperheight {11.69in}%
    \setlength\paperwidth  {8.27in}%
    \def\special@paper{8.5in,11in}
	\special{papersize=8.5in,11in}}

I see this evil is inherited from the IJCA template.  For a faq on paper size: https://texfaq.org/FAQ-papergeom

**Comment by matbesancon** [comment]
yes clearly, we didn't have time to change all the settings, better let them as-is for now :man_shrugging:
